uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -11,35 +11,32 @@ define([
   'leave-absences/shared/models/entitlement-model',
   'leave-absences/shared/models/leave-request-model',
   'leave-absences/shared/models/public-holiday-model',
-  'leave-absences/shared/models/instances/leave-request-instance',
-  'leave-absences/shared/models/instances/sickness-leave-request-instance',
-], function (components, _, moment) {
+], function (controllers, _, moment) {
```

Comment:
In this file `that = this` has been used many times, it should be removed and instead `bind(this)` can be used to the function to change context